packages/top-app-bar/index.js

```diff
@@ -69,10 +69,25 @@ export default class TopAppBar extends React.Component {
     this.foundation_.destroy();
   }
 
+  componentDidUpdate(prevProps) {
+    const foundationChanged = ['short', 'shortCollapsed', 'fixed']
+      .some((variant) => this.props[variant] !== prevProps[variant] );
```

Comment:
formatting issue, but for the sake of time I'm going to ignore this one and fix in a following pr